sdk/src/wallet/account/operations/address_generation.rs

```diff
@@ -88,14 +88,16 @@ impl Account {
                             self.emit(
                                 account_details.index,
                                 WalletEvent::LedgerAddressGeneration(AddressData {
-                                    address: address[0].to_bech32(bech32_hrp),
+                                    address: crate::types::block::address::ToBech32Ext::to_bech32(
+                                        address[0], bech32_hrp,
+                                    ),
```

Comment:
What's the reason for this change? Looks worse now

Reply:
Feature gated.